CHANGELOG

```diff
@@ -5,6 +5,12 @@ title: Details
 
 All notable changes to this project will be documented in this file.
 
+## security
+
+### 2020-07-30 v0.0.59
+Added
+- Notification callback after changes in policies / users / attributes
+
```

Comment:
It should be under the existing ## security section

Reply:
Done